app/src/main/res/values/strings.xml

```diff
@@ -63,111 +63,111 @@
     <string name="txt_autoplay_clips_after_loading_settings">Auto-play clip when it\'s loaded</string>
-    <string name="toast_autoplay_clip_on">Auto-play clip turned on</string>
-    <string name="toast_autoplay_clip_off">Auto-play clip turned off</string>
+    <string name="toast_autoplay_clip_on">Auto-play clip is now turned on.</string>
+    <string name="toast_autoplay_clip_off">Auto-play clip is now turned off.</string>
     <string name="txt_dark_theme_settings">Dark theme</string>
-    <string name="toast_dark_theme_on">Dark theme turned on</string>
-    <string name="toast_dark_theme_off">Dark theme turned off</string>
+    <string name="toast_dark_theme_on">Dark theme is now turned on.</string>
+    <string name="toast_dark_theme_off">Dark theme is now turned off.</string>
     <string name="contact_developer_on_telegram">Contact the developer on Telegram</string>
     <string name="txt_experimental_features">Experimental features</string>
     <string name="toast_experimental_features_on">Experimental features turned on\nNew features could be unstable and unsafe</string>
     <string name="toast_experimental_featured_off">Experimental features turned off</string>
-    <string name="txt_log_on_file">Save logs on a file</string>
-    <string name="toast_log_on_file_on">Saving logs on a file turned on\nYou can attach the log.txt file when you report an issue\nRead more on Github "Saving the log"</string>
-    <string name="toast_log_on_file_off">Saving logs on a file turned off</string>
+    <string name="txt_log_on_file">Save logs to a file</string>
+    <string name="toast_log_on_file_on">Saving logs to a file turned on.\nYou can attach the log.txt file when you report an issue.\nRead more on GitHub: \"Saving the log\"</string>
+    <string name="toast_log_on_file_off">Saving logs to a file turned off.</string>
     <string name="txt_anonymous_statistics">Anonymous statistics</string>
-    <string name="toast_anonymous_statistics_on">Anonymous statistics turned on.\nWe don\'t collect your personal data. We will use statistics only to understand the amount of app users per day.</string>
+    <string name="toast_anonymous_statistics_on">Anonymous statistics turned on.\nWe don\'t collect your personal data. We only use statistics to see how many people use the app each day.</string>
     <string name="toast_anonymous_statistics_off">Anonymous statistics turned off.</string>
     <string name="txt_skip_recording_confirmation">Skip recording confirmation</string>
-    <string name="toast_skip_recording_confirmation_on">Skip recording confirmation turned on.\nNow you will no longer forced to listen your recording before sending it, so you can send directly.</string>
+    <string name="toast_skip_recording_confirmation_on">Skip recording confirmation turned on.\nYour recordings will be sent directly without the opportunity to listen to them first.</string>
```

Comment:
It's not correct. You can listen, but you don't HAVE. It won't send automatically.

Reply:
"[...] You will be able to send recordings without necessarily listening to them."
What do you think?

Reply:
Ah, I misunderstood this message.

Please replace this with: "Skip recording confirmation turned on. You will now be able to send recordings without having to listen to them first."